sunpy/io/srs.py

```diff
@@ -0,0 +1,124 @@
+"""
+This module implements SRS File Reader.
+"""
+__author__ = "Sudarshan Konge"
+__email__ = "sudk1896@gmail.com"
+
+from astropy.table import Table, Column, vstack
+import collections, calendar, re
+
+__all__ = ['read']
+
+def read(filepath):
+    """
+    Method for reading an SRS File.
+    """
+    File = open(filepath, 'r')
+    lines = list()
+    for line in File:
+        arr = line.split()
+        store = list()
+        for i in range(0, min(len(arr), 8)):
+            store.append(arr[i])
+        lines.append(store)
+
+    #Store table meta-data. Only first two lines
+    #of SRS text file store the meta-data.
+    meta_data = collections.OrderedDict()
+    regex = re.compile('[^a-zA-Z]')
+    try:
+        meta_data[regex.sub('', lines[0][0])] = lines[0][1]
+        meta_data[regex.sub('', lines[1][0])] = ' '.join(lines[1][1:])
+    except IndexError:
+        print ("SRS meta-data is not available.")
+
+    #Problem: An array of strings. We need to extract
+    #three tables from these I, IA and II.
+
+    table = list() #This table holds the three tables which we would later merge.
+    indices = list()
+    
+    for i in range(0, len(lines)):
+        if (lines[i][0][0] == 'I'):
+            indices.append(i)
+    indices.append(len(lines))
+    indices.sort()
```

Comment:
Remove it, it's just confusing to see that you want to sort something when it should be sorted by definition.  Unless... you think that's not the case, then explain it as a comment.